src/test/java/io/mosip/mimoto/controller/InjiControllerTest.java

```diff
@@ -153,7 +153,7 @@ static IssuerDTO getIssuerDTO(String issuerName) {
     public void getAllIssuersTest() throws Exception {
         IssuersDTO issuers = new IssuersDTO();
         issuers.setIssuers((List.of(getIssuerDTO("Issuer1"), getIssuerDTO("Issuer2"))));
-        Mockito.when(issuersService.getAllIssuers())
+        Mockito.when(issuersService.getAllIssuers(null))
```

Comment:
Add test when search string is passed and api returns filtered issuers

Reply:
added the test case for filtered issuers